packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.module.js

```diff
@@ -28,6 +28,6 @@ angular
   ])
   .config(routing)
   .component('pciProjectTrainingDashboardComponent', component)
-  .run(/* @ngTranslationsInject:json ./translations */);
+  .run(/* @ngTranslationsInject:json ./translations ../jobs/translations */);
```

Comment:
We tend to scope translations into its own AngularJS module (here: `ovhManagerPciTrainingDashboard`).

Can you avoid relying on translations located in parents translation folders?